packages/daimo-api/src/api/getAccountHistory.ts

```diff
@@ -198,12 +199,24 @@ export async function getAccountHistory(
     const landlineSessionKey = (await getLandlineSession(address)).key;
     landlineSessionURL = getLandlineURL(address, landlineSessionKey);
     landlineAccounts = await getLandlineAccounts(address);
-    const landlineTransfers = await getLandlineTransfers(address);
-    transferClogs = addLandlineTransfers(
-      landlineTransfers,
-      transferClogs,
-      chainConfig.daimoChain
-    );
+    // Support for displaying landline transfers in the mobile app was added
+    // in version 1.9.8 and doesn't support backcompat.
+    // Only add landline transfers if the app version is > 1.9.7
+    if (
+      version.appVersion != null &&
+      (version.appVersion.major > 1 ||
+        (version.appVersion.major === 1 && version.appVersion.minor > 9) ||
+        (version.appVersion.major === 1 &&
+          version.appVersion.minor === 9 &&
+          version.appVersion.patch > 6)) // TODO, change this to > 7 during 1.9.8 release
```

Comment:
hmm could simplify to `daimoVersion: string` and then, here

```
if (semver.gt(daimoVersion, "1.9.6")) { // ...
```

https://www.npmjs.com/package/semver